inc/deploycheck.class.php

```diff
@@ -96,12 +108,12 @@ static function getLabelForAType($type) {
     * @return array
     */
    static function getUnitLabel() {
-      return array(
-         "B"  => __("B", 'fusioninventory'),
-         "KB" => __("KiB", 'fusioninventory'),
-         "MB" => __("MiB", 'fusioninventory'),
-         "GB" => __("GiB", 'fusioninventory')
-      );
+      return [
+               "B"  => __("B", 'fusioninventory'),
```

Comment:
Would be better to use same units as in the core (Kio, Mio and so on). Maybe use directly Toolbox::getSize() instead of a local method?